apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTableFixture.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright 2018 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Data.CommonTesting;
+using System;
+using System.Threading;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    [CollectionDefinition(nameof(TransactionTableFixture))]
+    public class TransactionTableFixture : SpannerTableFixture, ICollectionFixture<TransactionTableFixture>
+    {
+        /// <summary>
+        /// A timestamp which was definitely after the table existed, but before any other test-specific keys were added.
+        /// </summary>
+        internal DateTime TimestampBeforeEntries { get; private set; }
+
+        /// <summary>
+        /// A reasonable staleness such that the table will exist this long before tests run, but shouldn't
+        /// have any entries for TestKey. (The population process pauses for at least this long, and this should
+        /// be longer than all the transaction tests take to run.)
+        /// </summary>
+        internal TimeSpan Staleness { get; } = TimeSpan.FromSeconds(30);
+
+        public TransactionTableFixture() : base("TxTable")
+        {
+        }
+
+        protected override void CreateTable()
+        {
+            ExecuteDdl($@"CREATE TABLE {TableName} (
+                              K                   STRING(MAX) NOT NULL,
+                              Int64Value          INT64,
+                              StringValue         STRING(MAX),
+                            ) PRIMARY KEY (K)");
+        }
+
+        protected override void PopulateTable(bool fresh)
+        {
```

Comment:
`fresh` isn't used; is that correct?

Reply:
Yes, the aim is that some tests may need to know whether this is a new database or not. Most don't care.

We may or may not change our minds about that. (No worries about versioning though.)